src/transit/tests.py

```diff
@@ -94,7 +97,10 @@ def generate_tests(base_url):
                 url=base_url[:-1].replace("https", "http") + ":5000",
             ),
         ),
-        Test(name="api/docs 200", request=Request(method="GET", url=base_url + "api/docs/")),
+        Test(
```

Comment:
you should add the black formatter so it doesn't undo the auto formatting from previous commits

Reply:
I installed the black formatter and am using it, but it still seems to be changing some spots of code. Is there any extra configuration I'm missing here?
<img width="565" alt="Screen Shot 2019-09-29 at 6 18 26 PM" src="https://user-images.githubusercontent.com/31456107/65840315-8a77b580-e2e5-11e9-98a4-edee742a6e77.png">

Reply:
Are you sure it's this extension: https://marketplace.visualstudio.com/items?itemName=joslarson.black-vscode

Reply:
Also check out ithaca-transit-tcat-microservice's readme! It explains how to set up black on VSCode: https://github.com/cuappdev/ithaca-transit-tcat-microservice

Reply:
Yes, I'm using that one. Does app dev use a different line length than the default of 88?

Reply:
oh good catch, yea try 120

Reply:
Done. Should I make another PR with the latest commit?